java/src/hdf/hdf5lib/H5.java

```diff
@@ -523,7 +523,28 @@ public synchronized static native int H5set_free_list_limits(int reg_global_lim,
      * @exception HDF5LibraryException
      *                - Error from the HDF-5 Library.
      **/
-    public synchronized static native void H5export_dataset(String file_export_name, String file_name,
+    public synchronized static native void H5export_dataset(String file_export_name, long file_id,
+            String object_path, int binary_order) throws HDF5LibraryException;
+
+    /**
+     * H5export_attribute is a utility function to save data in a file.
+     *
+     * @param file_export_name
+     *            The file name to export data into.
+     * @param dataset_id
+     *            The identifier of the dataset containing the attribute.
+     * @param object_path
+     *            The attribute to be exported.
+     * @param binary_order
+     *            99 - export data as text.
+     *            1 - export data as binary Native Order.
+     *            2 - export data as binary Little Endian.
+     *            3 - export data as binary Big Endian.
+     *
+     * @exception HDF5LibraryException
+     *                - Error from the HDF-5 Library.
+     **/
+    public synchronized static native void H5export_attribute(String file_export_name, long dataset_id,
```

Comment:
not yet, I started with 1.10 first (completing the original merge) and since that has different region specs was able to add it correctly. Will merge back up to develop.